packages/components/table/src/table/style-helper.ts

```diff
@@ -113,7 +113,12 @@ function useStyle<T>(
       layout.updateElsHeight()
     }
     layout.updateColumnsWidth()
-    requestAnimationFrame(syncPosition)
+
+    // When the test case is running, the context environment simulated by jsdom may have been destroyed,
+    // and window.requestAnimationFrame does not exist at this time.
+    if (typeof window !== 'undefined') {
+      requestAnimationFrame(syncPosition)
+    }
```

Comment:
Maybe line 127 should also be added.

Reply:
Line 127 is within onMounted() which shouldn't cause a debounce/delayed call after (after test env destroy).

Reply:
OK, you can install element-plus from this link to verify whether your local test can pass.

https://github.com/element-plus/element-plus/pull/19837#issuecomment-2651810474

Reply:
Thanks for the hint @btea! 

I've already tested the change locally by patching node_modules directly:
* increased the vitest delay between jsdom destroyed and vitest exits
* applied this change to the element-plus dist file :)

Local tests are passing 🥳